medusa/common.py

```diff
@@ -584,6 +584,23 @@ def statusFromName(name, anime=False):
     }
 
     @staticmethod
+    def wanted_quality_from_status(showObj, status):
+        """Return true if that quality is wanted based on show quality and episode status."""
+        cur_status, cur_quality = Quality.splitCompositeStatus(int(status) or -1)
+        allowed_qualities, preferred_qualities = Quality.splitQuality(showObj.quality)
+
+        if cur_status not in {WANTED, DOWNLOADED, SNATCHED, SNATCHED_PROPER}:
+            return False
+
+        if cur_status != WANTED:
+            if preferred_qualities:
+                if cur_quality in preferred_qualities:
+                    return False
+            elif cur_quality in allowed_qualities:
+                return False
+        return True
+
+    @staticmethod
```

Comment:
@pymedusa/developers wondering in UNKNOWN should be in this list? afaik if quality is unknown, ANY new quality is wanted right ?